commands.go

```diff
@@ -8,10 +8,81 @@ import (
 	"os"
 	"os/exec"
 	"path"
+	"runtime"
 )
 
 type ChrootEnterMethod int
 
+// Return the qemu-user executable name (without a -static suffix)
+// to run binaries of the given Debian architecture, or nil if the
+// current architecture can run those binaries without using qemu.
+func GetQemuUser(debianArch string) (string, error) {
+	switch debianArch {
```

Comment:
Please rework this to use table-like data structure instead of switch-case approach.

Reply:
Sorry, I don't actually know Go. If someone who knows the language well enough to design APIs can put together a better API, that would be much appreciated.